app/assets/javascripts/Components/Result/summary_panel.jsx

```diff
@@ -253,10 +253,8 @@ export class SummaryPanel extends React.Component {
     } else {
       let rows = this.props.graceTokenDeductions.flatMap(d => {
         return [
-          <tr key={d["users.user_name"]}>
-            <th colSpan={2}>
-              {`${d["users.user_name"]} - (${d["users.first_name"]} ${d["users.last_name"]})`}
-            </th>
+          <tr key={d["users.user_name"]} align="left">
```

Comment:
Please don't add formatting as an attribute, this should be added as part of the css. In this case especially, the align attribute is not supported by html5 and should not be used.

For this PR, please remove it and we can discuss whether we should adjust the alignment in a different PR later on.